packages/plugins/other/visitor-plugin-common/src/utils.ts

```diff
@@ -403,7 +403,9 @@ export function getPossibleTypes(schema: GraphQLSchema, type: GraphQLNamedType):
   return [];
 }
 
-export function hasConditionalDirectives(directives: readonly DirectiveNode[]): boolean {
+export function hasConditionalDirectives(directives: readonly DirectiveNode[] | null | undefined): boolean {
+  if (!directives) return false;
```

Comment:
How about the following?
```js
const CONDITIONAL_DIRECTIVES = ['skip', 'include']; // This can be global.
return directives?.some(directive => CONDITIONAL_DIRECTIVES.includes(directive?.name?.value));
```